packages/flutter/lib/src/material/radio.dart

```diff
@@ -240,8 +241,23 @@ class Radio<T> extends StatefulWidget {
   /// The color to use when this radio button is selected.
   ///
   /// Defaults to [ThemeData.toggleableActiveColor].
+  ///
+  /// If [fillColor] returns a non-null color in the [MaterialState.selected]
+  /// state, it will be used over this.
```

Comment:
Using "over this" here and elsewhere may not be clear to all readers. It might be clearer to say "instead of this color".

Reply:
Done.